src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -427,9 +427,20 @@ String produceDts(Depgraph depgraph) {
       String originalPath = compilerInput.getSourceFile().getOriginalPath();
       if (opts.skipEmitRegExp != null && Pattern.matches(opts.skipEmitRegExp, originalPath))
         continue;
-      transitiveProvides.addAll(compilerInput.getProvides());
+      Collection<String> inputProvides = compilerInput.getProvides();
+      Collection<String> filteredProvides = new ArrayList<>();
```

Comment:
This seems unrelated to the change?

Reply:
It is related. Its in the commit description

Reply:
The new version of closure started reporting new synthetic provides. If we dont filter them out we create a bunch of garbage goog: module exports.